plotly/tools.py

```diff
@@ -386,23 +386,20 @@ def mpl_to_plotly(fig, resize=False, strip_style=False, verbose=False):
 
 ### graph_objs related tools ###
 
-# TODO: Scale spacing based on number of plots and figure size
-def get_subplots(rows=1, columns=1, horizontal_spacing=0.1,
-                 vertical_spacing=0.15, print_grid=False):
+def get_subplots(rows=1, columns=1, print_grid=False, **kwargs):
     """Return a dictionary instance with the subplots set in 'layout'.
 
     Example 1:
-        # stack two subplots vertically
-        fig = tools.get_subplots(rows=2)
-        fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x1', yaxis='y1')]
-        fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x2', yaxis='y2')]
+    # stack two subplots vertically
+    fig = tools.get_subplots(rows=2)
+    fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x1', yaxis='y1')]
+    fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x2', yaxis='y2')]
 
```

Comment:
Maybe changed to:

>    # stack two subplots vertically
>    fig = tools.get_subplots(rows=2)
>    fig.append_trace(Scatter(x=[1,2,3], y=[2,1,2]), row=1, col=1)
>  # alternatively: fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x1', yaxis='y1')]
>    fig.append_trace(Scatter(x=[1,2,3], y=[2,1,2]), row=2, col=1) 
>  # alternatively: fig['data'] += [Scatter(x=[1,2,3], y=[2,1,2], xaxis='x2', yaxis='y2')]

Not sure about the "alternatively" text - thoughts? @theengineear , @etpinard